config/ember-try.js

```diff
@@ -1,62 +1,68 @@
-'use strict';
+"use strict";
 
-const getChannelURL = require('ember-source-channel-url');
+const getChannelURL = require("ember-source-channel-url");
+const typeTests = require("./ember-try-typescript");
 
-module.exports = async function() {
+const typeScriptScenarios = typeTests.scenarios.map((s) => ({
+  ...s,
+  command: typeTests.command,
+}));
+
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-2.18',
+        name: "ember-lts-2.18",
```

Comment:
Lets avoid rewriting this file with prettier + double quotes (only to change subsequently to single quotes in a follow up).

Reply:
Dagnabbit, I though I’d avoided committing any of those. Reverting.

Reply:
Fixed, force pushed.